prusti-tests/tests/cargo_verify/test_map/.gitignore

```diff
@@ -0,0 +1 @@
+/prusti-*
```

Comment:
I think it would be better to remove all `cargo_verify` tests and instead have tests in `prusti-tests/tests/verify_overflow/fail/core_proof/mathematical_types/`.